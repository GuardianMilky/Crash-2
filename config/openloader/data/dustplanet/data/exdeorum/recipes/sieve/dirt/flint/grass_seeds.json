{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:flint_mesh","result":"exdeorum:grass_seeds","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.15}}