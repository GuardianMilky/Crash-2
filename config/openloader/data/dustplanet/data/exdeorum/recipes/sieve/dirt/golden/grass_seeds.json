{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:dirt"},"mesh":"exdeorum:golden_mesh","result":"exdeorum:grass_seeds","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.25}}